src/main/java/app/entities/unitconverter/LengthConverter.java

```diff
@@ -0,0 +1,36 @@
+package app.entities.unitconverter;
+
+public class LengthConverter
+{
+    public double meterToKilometer(double meter)
+        {
```

Comment:
this indentation is something that is usually addressed in these PRs.

We did however not settle on a code convention, but when we do this will not be it 😉